llmproxy/tests/unit_tests/llmproxy_client/test_llmproxy.py

```diff
@@ -11,20 +11,34 @@
 
 
 CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+path_to_env_test = ".env.test"
 
 
 # TODO: FIX
 def test_empty_model() -> None:
-    with pytest.raises(Exception, match="No models provided in user_settings"):
-        settings = _get_settings_from_yml(
-            path_to_yml=f"{CURRENT_DIRECTORY}/empty_model_test.yml"
-        )
-        available_model = _setup_available_models(settings=settings)
-        user_model = _setup_user_models(
-            settings=settings, available_models=available_model
+    # This need to be the specific exception that it would raise
+    # You need to merge main for the updated exceptions
+    with pytest.raises(Exception, match="No models provided in llmproxy.config.yml"):
+        # You need to use this
+        LLMProxy(
+            path_to_user_configuration=f"{CURRENT_DIRECTORY}/empty_model_test.yml",
+            path_to_env_vars=".env.test"
         )
 
 
+        # NOTE: These are technically class functions; You should not be using them directly like this
```

Comment:
Keep note of this; What excatly with this unit test? The use of the LLMProxy constructor or the individual functions?
You can of course test individual functions, but not like this; They need to be isolated and atomic, hence unit tests;

Reply:
Fix this issue in all the tests; Remember you are testing specific functionality; You are not trying to recreate the functionality in your unit and test that; At that point you are tesing your code, not the functionality in the code that exists.